docs/development/issues.md

```diff
@@ -30,6 +37,9 @@ contributing, and more. Please use the issue tracker for bugs only!
 
 ## Submitting a Bug Report
 
+To submit a bug report:
+[open an issue](https://github.com/electron/electronjs.org/issues/new).
```

Comment:
I don't think there needs to be a new paragraph for this.

Reply:
PR updated as recommended